src/main/java/org/scenarioo/api/util/xml/ScenarioDocuXMLFileUtil.java

```diff
@@ -77,15 +80,27 @@
 			}
 		}
 	}
-	
+
+	public static <T> List<T> unmarshalListOfFilesIgnoreInvalid(final Class<T> targetClass, final List<File> files) {
+		List<T> result = new ArrayList<T>();
+		for (File file : files) {
+			try {
+				result.add(unmarshal(targetClass, file));
+			} catch (Throwable e) {
+				LOGGER.error("Could not parse invalid File: " + file.getAbsolutePath() + " - will ignore this object.");
```

Comment:
Is "error" the right level? I would use "warn".

Reply:
I think there was something really completely wrong, usually I would throw an exception, but in that case it would block all other correct branches/builds from being loaded, so I thougth to put it to error is correct.